schema/F24.json

```diff
@@ -0,0 +1,365 @@
+    "version": "8.3.0.1",
+    "title": "FEC F24",
+    "description": "FORM 24 - 24 / 48 HOUR NOTICE OF INDEPENDENT EXPENDITURE",
+    "type": "object",
+    "required": [
+        "form_type",
+        "filer_committee_id_number",
+        "report_type_{24_48_hour}",
+        "treasurer_last_name",
+        "treasurer_first_name",
+        "date_signed"
+    ],
+    "fec_recommended": [
+        "committee_name",
+        "street_1",
+        "city",
+        "state",
+        "zip"
+    ],
+    "properties": {
+        "form_type": {
+            "title": "FORM TYPE",
+            "description": "",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 4,
+            "pattern": "^[ -~]{0,4}$",
+            "examples": [
+                "F24N"
+            ],
+            "fec_spec": {
+                "COL_SEQ": 1,
+                "FIELD_DESCRIPTION": "FORM TYPE",
+                "TYPE": "A/N-4",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "F24N",
+                "VALUE_REFERENCE": "F24+[N|A]",
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "filer_committee_id_number": {
+            "title": "FILER COMMITTEE ID NUMBER",
+            "description": "",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 9,
+            "pattern": "^[ -~]{0,9}$",
+            "examples": [
+                "C00123456"
+            ],
+            "fec_spec": {
+                "COL_SEQ": 2,
+                "FIELD_DESCRIPTION": "FILER COMMITTEE ID NUMBER",
+                "TYPE": "A/N-9",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "C00123456",
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": "this is the ID of the Committee Account the report/transaction is associated with",
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "report_type": {
+            "title": "REPORT TYPE",
+            "description": "",
+            "enum":[
+                "24",
+                "48"
+            ],
+            "examples": [
+                48
+            ],
+            "fec_spec": {
+                "COL_SEQ": 3,
+                "FIELD_DESCRIPTION": "REPORT TYPE {24/48 Hour}",
+                "TYPE": "A/N-2",
+                "REQUIRED": "X(error)",
+                "SAMPLE_DATA": 48,
+                "VALUE_REFERENCE": "24, 48",
+                "RULE_REFERENCE": "Error if Code is missing;\nError if Coded incorrectly.",
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "original_amendment_date": {
+            "title": "ORIGINAL AMENDMENT DATE",
+            "description": "",
+            "type": [
+                "string",
+                "null"
+            ],
+            "minLength": 0,
+            "maxLength": 8,
+            "pattern": "^\\d{0,8}$",
```

Comment:
We've been using this pattern for dates:

"type": "string",
            "minLength": 10,
            "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$",